src/popup/components/Modal.vue

```diff
@@ -109,11 +111,16 @@ export default defineComponent({
       }
     });
 
+    const handleClose = () => {
```

Comment:
We agreed within the team to use regular functions (not arrows) when setting up the functions. This makes the setup block easier to follow with eyes and distinguish functions from constants.